src/org/jitsi/service/neomedia/RawPacket.java

```diff
@@ -13,10 +13,10 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.jitsi.impl.neomedia;
+package org.jitsi.service.neomedia;
 
 
-import org.jitsi.service.neomedia.*;
+import org.jitsi.impl.neomedia.*;
```

Comment:
@damencho would it make sense to use `org.jitsi.service...ByteArrayBuffer`? or does that break stuff?